proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -330,39 +330,37 @@ Property wrappers can opt into passing a projected-value argument to a property-
 
 This method of initialization is not mandatory for functions using supported wrapper types, and it can be disabled by providing arguments in the wrapper attribute, including empty attribute arguments: `func log(@Traceable() _ value: Value) { ... }`.
 
-#### Arguments in the property-wrapper attribute
+##### Arguments in the property-wrapper attribute
 
-Arguments in the property-wrapper attribute as well as other default arguments to `init(wrappedValue:)` and `init(projectedValue:)` have the same semantics as default arguments. These arguments are injected into the initializer call at the call-site, and therefore are evaluated by the caller. The same restrictions applied to default arguments are also applied to property-wrapper attribute arguments.
+Arguments in the property-wrapper attribute as well as other default arguments to `init(wrappedValue:)` and `init(projectedValue:)` have the same semantics as default arguments. These arguments are injected into the initializer call at the call-site, and therefore are evaluated by the caller. The same restrictions applied to default arguments are also applied to attribute arguments of API property wrappers attached to parameters.
 
-#### Overload resolution of backing property-wrapper initializer
+##### Overload resolution of backing property-wrapper initializer
 
-Since the property wrapper is initialized at the call-site, the argument type can impact overload resolution of `init(wrappedValue:)` and `init(projectedValue:)`. For example, if a property wrapper defines overloads of `init(wrappedValue:)` with different generic constraints and that wrapper is used on a function parameter, e.g.:
+Though the property wrapper is initialized at the call-site, the argument type _cannot_ impact overload resolution of `init(wrappedValue:)` and `init(projectedValue:)`. For example, if a property wrapper defines overloads of `init(wrappedValue:)` with different generic constraints and that wrapper is used on a function parameter, e.g.:
 
 ```swift
-@propertyWrapper
-struct Wrapper<Value> {
+struct LateInitialized<Value> {
 
-  init(wrappedValue: Value) { ... }
+  init(wrappedValue: @escaping () -> Value) { ... }
 
-  init(wrappedValue: Value) where Value : Collection { ... }
+  init(wrappedValue: @escaping () -> Value) where Value == Void { ... }
```

Comment:
```suggestion
  init(wrappedValue: @escaping @autoclosure () -> Value) where Value == Void { ... }
```

Reply:
I fixed the typo here, but I actually think this section should be lifted out from the API wrapper section because it applies to both kinds of property wrappers